src/nn/ops/batch_norm.ts

```diff
@@ -15,6 +15,7 @@ export class BatchNormalization extends SingleOutputOperation implements
   private axis_?: number;
   private epsilon_?: number;
   private activation_?: MLOperator;
+  private needCheckOutputShape_ = true;
```

Comment:
Many ops defined this variable, can it be defined in parent class such as SingleOutputOperation?

Reply:
Actually, it's good to add this member to its base class. Here, it's on my purpose to remind we need use this `needCheckOutputShape_ ` member to check output shape. @mingmingtasd WDYT?

Reply:
I think code like line103: this.needCheckOutputShape_ = false already remind ,WDYT?